### ISSUE TITLE
support for {{ content_checkpoint }}

### ISSUE THREAD

**Issue by builtbylane** [Shopify/theme-check#763]
**Is your feature request related to a problem? Please describe.**
`{{ content_checkpoint }}` is required for checkpoint.liquid file on plus stores

Theme check should not warn: 

```shell
Unknown object 'content_checkpoint' used.theme-check[UndefinedObject](https://shopify.dev/docs/themes/tools/theme-check/checks/undefined-object)
```

**Describe the solution you'd like**
content_checkpoint added to allowed vars
